### PR TITLE
Add Title to encodable output

### PR DESCRIPTION
**Pull request by TheNeikos**
The 'title' field was missing

### CODE REVIEW

**Comment by kstep** [comment]
Please update fields numbering for consistency. Thank you.

**Comment by TheNeikos** [comment]
Done!

File: src/song.rs [hunk 1]
```diff
@@ -118,8 +118,9 @@ impl Encodable for Song {
         e.emit_struct("Song", 7, |e| {
```
Comment (kstep) [inline src/song.rs]: Please set number of fields to 8. Otherwise I'm ready to accept the PR. Thank you!

Comment (TheNeikos) [inline src/song.rs]: It is pretty visible that I haven't written these things by hand so far :smile:

**Comment by kstep** [comment]
LGTM! Thank you for your effort!

**Pull request merged**